assets/js/components/KeyMetrics/MetricsSelectionPanel/index.js

```diff
@@ -35,39 +35,63 @@ import {
 	KEY_METRICS_SELECTION_FORM,
 	KEY_METRICS_SELECTION_PANEL_OPENED_KEY,
 } from '../constants';
+import useViewOnly from '../../../hooks/useViewOnly';
 import SideSheet from '../../SideSheet';
 import Header from './Header';
 import Footer from './Footer';
 import Metrics from './Metrics';
 const { useSelect, useDispatch } = Data;
 
 export default function MetricsSelectionPanel() {
+	const isViewOnly = useViewOnly();
+
 	const isOpen = useSelect( ( select ) =>
 		select( CORE_UI ).getValue( KEY_METRICS_SELECTION_PANEL_OPENED_KEY )
 	);
-	const keyMetrics = useSelect( ( select ) => {
+	const savedMetrics = useSelect( ( select ) => {
```

Comment:
The name here seems a bit off since saved metrics are essentially what is returned by `getKeyMetrics` but this does a fair bit more. It kind of seems like the intersection of saved + viewable? 🤔 Maybe `savedViewableMetrics`? 😄

Reply:
Updated, thank you!